sparrow-py/pysrc/sparrow_py/expr.py

```diff
@@ -287,6 +287,9 @@ def is_not_null(self) -> "Expr":
         """Return a boolean expression indicating if the expression is not null."""
         return Expr.call("is_valid", self)
 
+    def collect(self, max: Optional[int] = None, window: Optional[Window] = None) -> "Expr":
+        return _collect(self, max, window)
```

Comment:
I think we can generalize the helper, so this could be `_aggregation("collect", self, window, max)`.

Reply:
I intentionally didn't to not associate collect with windows, but I can maybe rename to a helper?

Reply:
Actually -- with the `max` parameter, wouldn't that add an additional argument to the aggregation calls?